source/frontend/package.json

```diff
@@ -141,6 +133,9 @@
   "jest": {
     "resetMocks": false,
     "clearMocks": true,
+    "moduleNameMapper": {
+      "\\.(css|less|sass|scss)$": "<rootDir>/src/mocks/styleMock.js"
+    },
```

Comment:
I had to add this configuration to avoid Jest getting stuck with a few of our files that import css or scss directly; ie

```tsx
// fake example
import 'button.css';

export class Button {
...
}
```

We still have a few of these. Why it wasn't an issue before? beats me